### ISSUE TITLE
get_multi is not typed

### ISSUE THREAD

**Issue by jakaline-dev** [igorbenav/fastcrud#123]
**Describe the bug or question**
This is both an issue and a suggestion that will make breaking changes.

Issue: The returned dict from get_multi is not typed properly. For example,

```python
artist_urls = await artist_url_crud.get_multi(
    db=db_session,
    limit=None,
    return_total_count=False,
    return_as_model=True,
    schema_to_select=ArtistURL,
    artist_id=artist.id,
    source_id=None,
)
for artist_url in artist_urls["data"]: #artist_url is not typed as list[ArtistURL] but 'any'
```

Request: Also, when return_total_count is False and return_as_model is True, I don't see any reason why the returned object has to be wrapped with a 'data' key. While this works great when returned with FastAPI's API endpoint directly, I also use it in a lot of async functions and this is bugging me.

```python
artist_urls = await artist_url_crud.get_multi(
    db=db_session,
    limit=None,
    return_total_count=False,
    return_as_model=True,
    schema_to_select=ArtistURL,
    artist_id=artist.id,
    source_id=None,
)
for artist_url in artist_urls: # Why can we not do this?
```

**Comment by igorbenav** [comment]
> **Describe the bug or question** This is both an issue and a suggestion that will make breaking changes.
> 
> Issue: The returned dict from get_multi is not typed properly. For example,
> 
> ```python
> artist_urls = await artist_url_crud.get_multi(
>     db=db_session,
>     limit=None,
>     return_total_count=False,
>     return_as_model=True,
>     schema_to_select=ArtistURL,
>     artist_id=artist.id,
>     source_id=None,
> )
> for artist_url in artist_urls["data"]: #artist_url is not typed as list[ArtistURL] but 'any'
> ```

I'll work on it

**Comment by igorbenav** [comment]
> Request: Also, when return_total_count is False and return_as_model is True, I don't see any reason why the returned object has to be wrapped with a 'data' key. While this works great when returned with FastAPI's API endpoint directly, I also use it in a lot of async functions and this is bugging me.
> 
> ```python
> artist_urls = await artist_url_crud.get_multi(
>     db=db_session,
>     limit=None,
>     return_total_count=False,
>     return_as_model=True,
>     schema_to_select=ArtistURL,
>     artist_id=artist.id,
>     source_id=None,
> )
> for artist_url in artist_urls: # Why can we not do this?
> ```

What exactly is the issue of doing 

```python
for artist_url in artist_urls["data"]
```

instead?

I guess we could add a parameter for raw returns, but it will add some complexity to the code when a simple `artist_urls["data"]` could be used

**Comment by igorbenav** [comment]
Closed by #190
